pkg/types/powervs/defaults/platform.go

```diff
@@ -25,3 +25,13 @@ func SetPlatformDefaults(p *powervs.Platform) {
 	// MustParseCIDR parses a CIDR from its string representation. If the parse fails, the function will panic.
 	DefaultMachineCIDR = ipnet.MustParseCIDR(fmt.Sprintf("192.168.%d.0/24", subnet))
 }
+
+// DefaultNTPServer returns the NTP server provided for PowerVS.
+func DefaultNTPServer() string {
```

Comment:
Can we make do with constants or do we need methods for these values?

Reply:
@sadasu we were looking for the best practice here. Originally we had it hard coded in the manifest structure. A constant would do as well, what do you prefer?